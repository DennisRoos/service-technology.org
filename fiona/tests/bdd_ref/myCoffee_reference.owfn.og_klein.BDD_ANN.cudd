.ver DDDMP-2.0
.mode A
.varinfo 4
.dd bddAnn
.nnodes 15
.nvars 10
.nsuppvars 10
.suppvarnames !coffee !euro !tea ?beverage ?money final 0 1 2 3
.orderedvarnames 2 1 0 !coffee !tea ?beverage !euro ?money final 3
.ids 0 1 2 3 4 5 6 7 8 9
.permids 3 6 4 5 7 8 2 1 0 9
.nroots 1
.rootids -15
.nodes
1 1 0 0
2 9 1 -1
3 7 2 1
4 6 2 1
5 2 3 4
6 8 2 1
7 2 3 6
8 1 5 7
9 5 6 1
10 2 9 3
11 4 1 -4
12 3 1 11
13 2 4 -12
14 1 10 13
15 0 8 14
.end
