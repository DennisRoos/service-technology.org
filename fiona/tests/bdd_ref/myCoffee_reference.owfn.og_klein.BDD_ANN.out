digraph "DD" {
size = "7.5,10"
center = true;
edge [dir = none];
{ node [shape = plaintext];
  edge [style = invis];
  "CONST NODES" [style = invis];
" 0 " -> " 1 " -> " 2 " -> " 3 " -> " 4 " -> " 5 " -> " 6 " -> " 7 " -> " 8 " -> " 9 " -> "CONST NODES"; 
}
{ rank = same; node [shape = box]; edge [style = invis];
"F0"; }
{ rank = same; " 0 ";
"22a";
}
{ rank = same; " 1 ";
"225";
"21b";
}
{ rank = same; " 2 ";
"21d";
"21e";
}
{ rank = same; " 3 ";
"21a";
"233";
"248";
"209";
}
{ rank = same; " 4 ";
"247";
"22d";
"242";
"1fc";
"219";
"214";
"208";
"232";
}
{ rank = same; " 5 ";
"244";
"205";
"246";
"213";
"207";
"20e";
"241";
"218";
"231";
"226";
"1fa";
"23e";
"22c";
"216";
"25d";
"22f";
}
{ rank = same; " 6 ";
"20b";
"25c";
"1fd";
"22e";
"20d";
"215";
"212";
"217";
"224";
"206";
"240";
"230";
"22b";
"25a";
"210";
"1f6";
"243";
"239";
"238";
"1f4";
"245";
"227";
}
{ rank = same; " 7 ";
"25e";
"20f";
"211";
"221";
"1f5";
"228";
"23f";
"229";
"259";
"25b";
"20a";
"223";
"234";
"20c";
"237";
"1f2";
}
{ rank = same; " 8 ";
"222";
"21c";
"256";
"220";
}
{ rank = same; " 9 ";
"21f";
}
{ rank = same; "CONST NODES";
{ node [shape = box]; "1e4";
}
}
"F0" -> "22a" [style = dotted];
"22a" -> "21b";
"22a" -> "225" [style = dashed];
"225" -> "21d";
"225" -> "21e" [style = dashed];
"21b" -> "209";
"21b" -> "21a" [style = dashed];
"21d" -> "209";
"21d" -> "233" [style = dashed];
"21e" -> "21a";
"21e" -> "248" [style = dashed];
"21a" -> "214";
"21a" -> "219" [style = dashed];
"233" -> "22d";
"233" -> "232" [style = dashed];
"248" -> "242";
"248" -> "247" [style = dashed];
"209" -> "1fc";
"209" -> "208" [style = dashed];
"247" -> "244";
"247" -> "246" [style = dashed];
"22d" -> "226";
"22d" -> "22c" [style = dashed];
"242" -> "23e";
"242" -> "241" [style = dashed];
"1fc" -> "25d";
"1fc" -> "1fa" [style = dashed];
"219" -> "216";
"219" -> "218" [style = dashed];
"214" -> "20e";
"214" -> "213" [style = dashed];
"208" -> "205";
"208" -> "207" [style = dashed];
"232" -> "22f";
"232" -> "231" [style = dashed];
"244" -> "243";
"244" -> "239" [style = dashed];
"205" -> "1fd";
"205" -> "25c" [style = dashed];
"246" -> "245";
"246" -> "1e4" [style = dashed];
"213" -> "210";
"213" -> "212" [style = dashed];
"207" -> "206";
"207" -> "1f6" [style = dashed];
"20e" -> "20b";
"20e" -> "20d" [style = dashed];
"241" -> "240";
"241" -> "1e4" [style = dashed];
"218" -> "217";
"218" -> "212" [style = dashed];
"231" -> "230";
"231" -> "22b" [style = dashed];
"226" -> "21f";
"226" -> "224" [style = dashed];
"1fa" -> "1f4";
"1fa" -> "1f6" [style = dashed];
"23e" -> "238";
"23e" -> "239" [style = dashed];
"22c" -> "227";
"22c" -> "22b" [style = dashed];
"216" -> "215";
"216" -> "20d" [style = dashed];
"25d" -> "25a";
"25d" -> "25c" [style = dashed];
"22f" -> "22e";
"22f" -> "224" [style = dashed];
"20b" -> "234";
"20b" -> "20a" [style = dashed];
"25c" -> "221";
"25c" -> "25b" [style = dashed];
"1fd" -> "221";
"1fd" -> "259" [style = dashed];
"22e" -> "221";
"22e" -> "21f" [style = dashed];
"20d" -> "220";
"20d" -> "20c" [style = dashed];
"215" -> "220";
"215" -> "20a" [style = dashed];
"212" -> "1e4";
"212" -> "211" [style = dashed];
"217" -> "1e4";
"217" -> "20f" [style = dashed];
"224" -> "221";
"224" -> "223" [style = dashed];
"206" -> "228";
"206" -> "25e" [style = dashed];
"240" -> "1f2";
"240" -> "23f" [style = dashed];
"230" -> "228";
"230" -> "221" [style = dashed];
"22b" -> "228";
"22b" -> "229" [style = dashed];
"25a" -> "21f";
"25a" -> "259" [style = dashed];
"210" -> "1f2";
"210" -> "20f" [style = dashed];
"1f6" -> "228";
"1f6" -> "1f5" [style = dashed];
"243" -> "220";
"243" -> "237" [style = dashed];
"239" -> "220";
"239" -> "1f2" [style = dashed];
"238" -> "234";
"238" -> "237" [style = dashed];
"1f4" -> "222";
"1f4" -> "25e" [style = dashed];
"245" -> "1e4";
"245" -> "23f" [style = dashed];
"227" -> "222";
"227" -> "221" [style = dashed];
"25e" -> "220";
"25e" -> "256" [style = dashed];
"20f" -> "1e4";
"20f" -> "256" [style = dashed];
"211" -> "1e4";
"211" -> "21c" [style = dashed];
"221" -> "220";
"221" -> "21f" [style = dashed];
"1f5" -> "220";
"1f5" -> "21c" [style = dashed];
"228" -> "1e4";
"228" -> "222" [style = dashed];
"23f" -> "1e4";
"23f" -> "220" [style = dashed];
"229" -> "220";
"229" -> "222" [style = dashed];
"259" -> "21f";
"259" -> "256" [style = dashed];
"25b" -> "21f";
"25b" -> "21c" [style = dashed];
"20a" -> "222";
"20a" -> "256" [style = dashed];
"223" -> "21f";
"223" -> "222" [style = dashed];
"234" -> "21f";
"234" -> "220" [style = dashed];
"20c" -> "222";
"20c" -> "21c" [style = dashed];
"237" -> "222";
"237" -> "220" [style = dashed];
"1f2" -> "222";
"1f2" -> "1e4" [style = dashed];
"222" -> "1e4";
"222" -> "21f" [style = dashed];
"21c" -> "1e4";
"21c" -> "1e4" [style = dotted];
"256" -> "21f";
"256" -> "1e4" [style = dotted];
"220" -> "21f";
"220" -> "1e4" [style = dashed];
"21f" -> "1e4";
"21f" -> "1e4" [style = dotted];
"1e4" [label = "1"];
}
